### DISCUSSION TITLE
Figure rendered inside a table in MS Word when using fig-cap

### DISCUSSION THREAD

**Discussion by jtrecenti** [quarto-dev/quarto-cli#3999] [Q&A] [answered]
Hi! Thank you very much for developing Quarto. It is awesome.

I was trying to tweak table styles in MS Word (painful, but needed) when I realized that this code:

```
---
title: "Untitled"
format: docx
---

Lets's see @fig-aaa.

```{r}
#| label: fig-aaa
#| fig-cap: a nice figure.
plot(1:10)
```

generates this document:

![image](https://user-images.githubusercontent.com/1425970/213029492-7b20fb86-3247-4baf-9f60-700f2b276a63.png)

Note the "+" sign on the top left of the image and the table styling options at the top menu, indicating that we are inside a table.

So, apparently, when we use `fig-cap:`, it wraps the figure and the caption inside a 1-cell table. 

Two questions

1. Is this the desired behavior?
2. Is it possible to turn it off?

Thanks!

**Comment by cscheid** [comment, marked as answer]
This is a present limitation of the way we emit captions and figures. It's also currently hard-coded.

What would you like to replace it with?

**Reply by jtrecenti** [reply]
The problem I'm facing is that I want to edit table default styling, like borders. However, when I do this, the border is added to the image also, which is not the desired result.

Anyway, It's good to know that it is a limitation and I'm not crazy! For now I'll just drop the table borders. Thanks!

**Reply by cscheid** [reply]
Ok, thanks for the followup. That's a totally valid reason to want to change it! I'm going to flag it as an enhancement request. I can't give you a time when we'll be able to do better, but we are going through a redesign that might enable us to fix this.
